subcommittees/mentoring/procedure-debriefing-session.md

```diff
@@ -0,0 +1,93 @@
+# Procedure - Debriefing Session
+
+This document intends to describe how debriefing sessions
+are organized so that it continue even after members go away.
+
+## Terminology
+
+-   **Debriefing session**: is an online meeting
+    where instructors could share experience about the workshops
+    they taught.
+-   **Facilitator**: is one member of the Mentoring Subcommittee
+    that attend the debriefing session to coordinate it
+    and write one blog post about it.
+
+## We Need a Debriefing Session
+
+1.  SCF is a community based project
+    and the community need opportunity to interact.
+
+2.  SCF believe in jugyokenkyu, or "lesson study",
+    and evidence-based education
+    so instructors need opportunity to change experience.
+
+## Let's do a Debriefing Session
+
+The Mentoring Subcommittee is responsible to organize
+the debriefing sessions so that
+
+-   each month has at least one debriefing session and
+
+-   instructors are notify with at least one week in advance.
+
+## Inviting Instructors
+
+The invitation should be send directly to instructors by email.
+The contact address of the instructors are available at
+https://amy.software-carpentry.org/workshops/debrief.
+
+Optionally,
+the invitation could be send to
+[Instructors mailing list](http://lists.software-carpentry.org/mailman/listinfo/instructors_lists.software-carpentry.org).
+
+## Show time
+
+Each debriefing session **must** have at least one **facilitator**.
+
+The **facilitator** is responsible to provide one place
+where instructors attending the debriefing could share notes
+of the debriefing.
+
+The **facilitator** is also responsible
+to help instructors answer the questions on the next session.
+
+## Questions
+
+1.  Public
+
+    1.  How many learners attend the workshop?
+
+    2.  Big changes in the number of learners during the workshop?
+
+    3.  Learners had any installation problem with the tools
+        requested that they installed?
+
+2.  Content
+
+    1.  What was actually taught?
+        Shell, Git, Python, R, MATLAB or SQL?
+
+    2.  How much time was spent on each topic?
+
+    3.  Did you adapt the lesson material or use it as written?
+
+3.  Feedback
+
+    1.  What worked well that we should do again?
+
+    2.  What went wrong / fell flat / shouldn't we ever, ever do again?
+
+4.  Extra
+
+    1.  Anything else that might be interesting?
+
+    2.  Who were helpers, and their contact info?
+        (especially those not listed on course website)
+
+## Report to the Community
+
+After each debriefing session,
+the **facilitator** of that debriefing session **must**
+
+1.  write a blog post summarizing the topics discussed and
+2.  create issues at the properly place for comments about the lessons.
```

Comment:
Is this a new policy? I remember that last time it came up the present facilitators didn't agree who's responsibility this is (facilitators or instructors).